src/scss/sections/_product-waitlist.scss

```diff
@@ -453,3 +453,32 @@
 		right: 8px;
 	}
 }
+
+.product-waitlist-form {
```

Comment:
we need to optimise the code above, please create the element in the DOCS with this version and another dev can work on it